packages/mui-codemod/codemod.js

```diff
@@ -83,6 +83,14 @@ async function runJscodeshiftTransform(transform, files, flags, codemodFlags) {
 
   if (jscodeshiftProcess.error) {
     throw jscodeshiftProcess.error;
+  } else if (!flags.skipPrettier) {
+    try {
+      childProcess.execSync('prettier --write $(git diff --name-only)', {
+        stdio: 'inherit',
+      });
+    } catch (error) {
+      // silentyly ignore errors
+    }
```

Comment:
@Janpot what do you think?

Reply:
> which can result in failures in the shell as there are limits to the command length

I think it's an edge case because the prettier only runs on changed files. I used to run codemod on several large projects (backstage is one of them) but I've never found a case where file changes go beyond hundreds before.

Anyway, in case there is an error, `--skip-prettier` can be used.

Reply:
What we could do is to not add prettier at first and see the feedback. I will remove it for now.

Reply:
I ended up here after running the deprecations codemod in our docs. I had some files present the [double parens issue](https://github.com/facebook/jscodeshift/issues/534). In my case, it was easy to bypass running prettier manually. It wasn't that bad. I wouldn't complain if prettier was ran automatically, maybe as an opt-in.

Reply:
We should at least add a warning for this in the [codemod readme](https://github.com/mui/material-ui/tree/next/packages/mui-codemod).